radish/parser.py

```diff
@@ -436,33 +453,45 @@ def _parse_skip_scenario(self, line):
 
         return True
 
+    def _detect_keyword(self, keyword, line):
+        """
+        Detects a keyword on a given line
+        :param keyword: the keyword to detect
+        :param line: the line in which we want to detect the keyword
+        :return: the line without the detected keyword
+        :rtype: string or None
+        """
+        pattern = "^%s\s*%s" % (keyword, self._keywords_delimiter)
+        match = re.match(pattern, line)
+
+        if match:
+            return line[line.index(self._keywords_delimiter) + 1:].strip()
```

Comment:
What about matching the sentence as a regex group instead of string slicing?

```python
pattern = "^%s\s*%s\s*(.*)$" % (keyword, self._keywords_delimiter)
...
```